easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-12.2.0.eb

```
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDB'
+version = '13.2'
+
+homepage = 'https://www.gnu.org/software/gdb/gdb.html'
+description = "The GNU Project Debugger"
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['fd5bebb7be1833abdb6e023c2f498a354498281df9d05523d8915babeb893f0a']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('makeinfo', '7.0.3'),
+]
+
+dependencies = [
+    ('zlib', '1.2.12'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.3'),
+    ('expat', '2.4.9'),
+    ('Python', '3.10.8'),
+    ('ISL', '0.26'),
+    ('MPC', '1.3.1'),
+]
+
+preconfigopts = "mkdir obj && cd obj && "
+configure_cmd_prefix = '../'
+prebuildopts = "cd obj && "
+preinstallopts = prebuildopts
+
+configopts = '--with-system-zlib --with-system-readline --with-expat=$EBROOTEXPAT '
+configopts += '--with-python=$EBROOTPYTHON/bin/python --with-isl=$EBROOTISL --with-mpc=$EBROOTMPC '
+configopts += '--enable-tui --enable-plugins --disable-install-libbfd '
+
+sanity_check_paths = {
+    'files': ['bin/gdb', 'bin/gdbserver'],
+    'dirs': [],
+}
+
```

Can we add some simple `sanity_check_commands` here?
```suggestion

sanity_check_commands = [
    'gdb --help',
    'gdbserver --help',
]

```

---

That is new, but why not. Makes sense I guess.